src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -49,18 +52,19 @@ internal void CalculateEffectivePackageSaveMode()
                 packageSaveModeValue = SettingsUtility.GetConfigValue(Settings, "PackageSaveMode");
             }
 
-            EffectivePackageSaveMode = PackageSaveModes.None;
             if (!string.IsNullOrEmpty(packageSaveModeValue))
             {
+                // Restore \ Install always extract files
+                EffectivePackageSaveMode = Packaging.PackageSaveMode.Files;
```

Comment:
There's never an option to not extract files when installing or restoring packages. The `Files` replaces the `extractManifestOnly` value that was passed in to `PackageExtractor`.